### PR TITLE
Add guidance for delivering feedback

### PR DESCRIPTION
**Pull request by NatashaMcCann**
WHY:

As per issue #207, the requester has advised that they would like more guidance on how to deliver feedback and how to write competencies. HR have written a guide on how to deliver great feedback, using whats called the AID model which may be a useful tool to use as a template when writing up individual competencies.

WHAT:

Updated the FAQ and the How To Use framework to give individuals looking at this clear guidance to follow when writing feedback on themselves or as managers feedback on their colleagues.

### CODE REVIEW

File: site/pages/faq.md [hunk 1]
```diff
@@ -82,3 +82,7 @@ No, competencies are not weighted. No competency is more important than any othe
 ## Who is in charge of maintaining the competencies?
 
 A group of engineers from across Product &amp; Technology. See [The Engineering Progression Working Group docs](/docs/working-group/), or check the [GitHub team](https://github.com/orgs/Financial-Times/teams/engineering-progression-working-group/members) for individual members.
+
+## Guidelines for writing feedback or writing about competencies you have achieved
```
Comment (rowanmanning) [inline site/pages/faq.md:86]: I think we should either remove this from the FAQ (just leaving it in the "how to use" guide), or rephrase this to be a question. So far it's not a question that we've been asked a lot, so maybe not including it in the FAQ is the way to go. Adding the rest of the group as a reviewer so they get notified.

Comment (rowanmanning) [reply site/pages/faq.md:86]: OK we agreed that rephrasing this as a question and leaving it in the FAQs

**Comment by dereknahman** [comment]
Hello. I've implemented the change, and this is ready for review ✨

**Review by rowanmanning — APPROVED** [review]
Looks good to me 👍 thanks!
